### ISSUE TITLE
Meaning of recommended properties on Standards

### ISSUE THREAD

**Issue by jakubklimek** [SEMICeu/GeoDCAT-AP#101]
**Motivation**
GeoDCAT-AP 2.0.0 specifies [recommended properties](https://semiceu.github.io/GeoDCAT-AP/releases/2.0.0/#recommended-properties-for-standard) for [Standards](https://semiceu.github.io/GeoDCAT-AP/releases/2.0.0/#properties-for-standard):
- [description](https://semiceu.github.io/GeoDCAT-AP/releases/2.0.0/#standard-description)
- [identifier](https://semiceu.github.io/GeoDCAT-AP/releases/2.0.0/#standard-identifier)
- [reference register](https://semiceu.github.io/GeoDCAT-AP/releases/2.0.0/#standard-reference-register)
- [release date](https://semiceu.github.io/GeoDCAT-AP/releases/2.0.0/#standard-release-date)
- [title](https://semiceu.github.io/GeoDCAT-AP/releases/2.0.0/#standard-title)
- [type](https://semiceu.github.io/GeoDCAT-AP/releases/2.0.0/#standard-type)
- [version](https://semiceu.github.io/GeoDCAT-AP/releases/2.0.0/#standard-version)

However, even in GeoDCAT-AP examples, the recommended properties are not used, e.g. [Example 25](https://semiceu.github.io/GeoDCAT-AP/releases/2.0.0/#ex-standard-crs):
```turtle
<http://www.opengis.net/def/crs/OGC/1.3/CRS84> a dct:Standard ;
  dct:identifier "urn:ogc:def:crs:OGC:1.3:CRS84"^^xsd:anyURI ;
  dct:title "CRS84"@en ;
  dct:type <http://inspire.ec.europa.eu/glossary/SpatialReferenceSystem> ;
  skos:inScheme <http://www.opengis.net/def/crs/OGC> ;
  skos:prefLabel "CRS84"@en .
```

**Questions**
1. What is the meaning of *recommended* here? Should the properties be used with all standards mentioned in GeoDCAT-AP? e.g. with `http://www.opengeospatial.org/standards/wms` in the following example?
```turtle
[] a dcat:DataService;
    dct:conformsTo <http://www.opengeospatial.org/standards/wms> .
```
2. Should they be used only for standards for which there are no URIs?
3. Is there a registry of standards in which the standards are documented using the recommended properties?

**Possible proposals**
1. Move recommended properties to optional properties for Standards

**Comment by jakubklimek** [comment]
Voting on this issue postponed to give the community more time to discuss it here in GitHub.

**Comment by jakubklimek** [comment]
Since no additional feedback was received, this is resolved in sync with #109 - all properties for supporting entities, including Standards, will be optional.
